includes/Hooks.php

```diff
@@ -374,7 +383,43 @@ public function onLocalUserCreated( $user, $autocreated ) {
 		// New users get echo preferences set that are not the default settings for existing users.
 		// Specifically, new users are opted into email notifications for thanks.
 		if ( !$user->isTemp() && !$autocreated ) {
-			$this->userOptionsManager->setOption( $user, 'echo-subscriptions-email-edit-thank', true );
+			$userOptionsManager = MediaWikiServices::getInstance()->getUserOptionsManager();
+			$userOptionsManager->setOption( $user, 'echo-subscriptions-email-edit-thank', true );
+		}
+	}
+
+	/**
+	 * Add thanks button to SpecialMobileDiff page
+	 * @param OutputPage &$output OutputPage object
+	 * @param MobileContext $ctx MobileContext object
+	 * @param array $revisions Array with two elements, either nulls or RevisionRecord objects for
+	 *     the two revisions that are being compared in the diff
+	 */
+	public static function onBeforeSpecialMobileDiffDisplay( &$output, $ctx, $revisions ) {
```

Comment:
check that because I don't know what that is, it's not in original commit https://github.com/Wikia/mediawiki-extensions-Thanks/pull/4/commits/dcbbee36601a823c8eb9e36f8f7c12c403ed1f69

Reply:
It was disabled in extension.json before https://github.com/wikimedia/mediawiki-extensions-Thanks/compare/REL1_39...Wikia:mediawiki-extensions-Thanks:REL1_39-Fandom-adjustments